hpdbv3/hpdb_v3_examples_test.go

```diff
@@ -1,7 +1,7 @@
 // +build examples
 
 /**
- * (C) Copyright IBM Corp. 2021.
+ * (C) Copyright IBM Corp. 2022.
```

Comment:
The license for updated file looks like below

* (C) Copyright IBM Corp. 2021, 2022